docs/OnlineEditor.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: online-editor
+title: Online Editor
+---
+
+You can also use the online editor (https://casbin.org/en/editor/) to write your Casbin model and policy in your web browser. It provides functionality such as ``syntax highlighting`` and ``code completion``, just like an IDE for a programming language.
+
+## Use Pattern
+If you use ``RBAC with pattern`` or ``RBAC with all pattern``, it specifies the pattern matching function in the lower left corner.
+
+![editor-tips](https://casbin.org/img/editor-tips.png). 
```

Comment:
Actually you should global search all terms of `casbin.org` and make sure they are properly handled